docs/content/preview/admin/yb-admin.md

```diff
@@ -1114,32 +1114,38 @@ $ curl -s http://<any-master-ip>:7000/cluster-config
 
 #### set_preferred_zones
 
-Sets the preferred availability zones (AZs) and regions.
+Sets the preferred availability zones (AZs) and regions. Tablet leaders are placed in alive and healthy nodes of the highest preference zones (preference 1). When no such node is available, then alive and healthy nodes from the next highest preferred zone are picked. Zones with no preference are equally eligible to host tablet leaders.
```

Comment:
Good clarification. Instead of saying "highest preference", perhaps re-word as "most preferred" ?  Something like .. "are placed in alive and healthy nodes in availability zone in order of preference" ?

Reply:
Tablet leaders are placed in alive and healthy nodes of AZs in order of preference. When no healthy node is available in the preferred AZ, then alive and healthy nodes from the next preferred zone are picked. Zones with no preference are equally eligible to host tablet leaders.